.github/workflows/schema-check.yml

```diff
@@ -63,24 +62,11 @@ jobs:
           pip install -r dev-requirements.txt -r editable-requirements.txt
           python scripts/collect-artifact-schema.py --path ${{ env.LATEST_SCHEMA_PATH }}
 
-      # Copy generated schema files into the schemas.getdbt.com repo
-      # Do a git diff to find any changes
-      # Ignore any date or version changes though
       - name: Compare schemas
         run: |
           cp -r ${{ env.LATEST_SCHEMA_PATH }}/dbt ${{ env.SCHEMA_REPO_DIRECTORY }}
           cd ${{ env.SCHEMA_REPO_DIRECTORY }}
-          diff_results=$(git diff -I='*[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T' \
-          -I='*[0-9]{1}.[0-9]{2}.[0-9]{1}(rc[0-9]|b[0-9]| )' --compact-summary)
-          if [[ $(echo diff_results) ]]; then
-            echo $diff_results
-            echo "Schema changes detected!"
-            git diff -I='*[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T' \
-              -I='*[0-9]{1}.[0-9]{2}.[0-9]{1}(rc[0-9]|b[0-9]| )' > ${{ env.SCHEMA_DIFF_ARTIFACT }}
-            exit 1
-          else
-            echo "No schema changes detected"
-          fi
+          git diff -I='*[0-9]{4}-[0-9]{2}-[0-9]{2}' -I='*[0-9]+\.[0-9]+\.[0-9]+' --exit-code > ${{ env.SCHEMA_DIFF_ARTIFACT }}
```

Comment:
brought in changes from dbt-core here: https://github.com/dbt-labs/dbt-core/blob/main/.github/workflows/schema-check.yml#L81-L89